Example-OSX/Pods/Local Podspecs/VOKBenkode.podspec

```diff
@@ -1,13 +1,14 @@
 Pod::Spec.new do |s|
   s.name             = "VOKBenkode"
-  s.version          = "0.1.2"
+  s.version          = "0.2.0"
   s.summary          = "An Objective-C library for encoding/decoding objects using Bencoding."
   s.homepage         = "https://github.com/vokalinteractive/VOKBenkode"
   s.license          = { :type => 'MIT', :file => 'LICENSE' }
   s.author           = { "VOKAL Interactive" => "hello@vokalinteractive.com" }
   s.source           = { :git => "https://github.com/vokalinteractive/VOKBenkode.git", :tag => s.version.to_s }
```

Comment:
Just making a note that you'll want to update these two lines when the rebrand is launched.